test/common/router/rds_impl_test.cc

```diff
@@ -464,18 +474,25 @@ name: foo
           *message_ptr);
   MessageUtil::loadFromYaml(R"EOF(
 static_route_configs:
-  - name: foo
-    virtual_hosts:
-      - name: bar
-        domains: ["*"]
-        routes:
-          - match: { prefix: "/" }
-            route: { cluster: baz }
+  - route_config:
```

Comment:
I think it would make sense to have an `ads_integration_test` `TEST_P` that covers two distinct config deliveries and a validation that the `last_updated` is at least the delta time between config updates.

Reply:
I think validating a config dump in a parameterized test will be hard, since the config dump checks tend to be hand-written. ( https://github.com/envoyproxy/envoy/blob/master/test/server/listener_manager_impl_test.cc#L405-L416 ) Is there another place we could check two updates across two managers?